src/main/java/core/basesyntax/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+    private static final int TITLE_INDEX = 0;
+    private List<String> lines = new ArrayList<>();
```

Comment:
this variable is redundant on class level position, move please it declaration to the method fileReader().

Reply:
In previous comment I meant `lines = new ArrayList<>();` 
`TITLE_INDEX = 0;` - is ok, move it back please.